src/commands/sendPost.js

```diff
@@ -0,0 +1,44 @@
+const inquirer = require('inquirer')
+const socialInboxApi = require('../api/socialInboxApi')
+const chalk = require('chalk')
+const config = require('../config/config')
+const { v4: uuidv4 } = require('uuid')
+
+async function sendPost (message) {
+  try {
+    console.log(chalk.blue('Sending a post to followers...'))
+
+    const actorUsername = config.actorUsername
+
+    if (!actorUsername) {
+      console.log(chalk.yellow('No actor registered. Please register an actor first using "register-actor" command.'))
+      return
+    }
+
+    const activityId = `${config.socialInboxUrl}/${encodeURIComponent(actorUsername)}/outbox/${uuidv4()}`
```

Comment:
Instead of generating a new activity we should have users pass a path to the activity that's published in the site via the DP API.

Alternately we should save both the Activity and Note to the DP published site as well as update the outbox. This would bring the functionality closer to the jekyll plugin.

Reply:
```suggestion
    const activityId = activity.id
```